src/api/wrappers/cf_wrapper.py

```diff
@@ -81,3 +81,16 @@ def get_mods(self, mod_ids: list[int]) -> list[CFMod]:
                 "filterPcOnly": True
             },
             unpacker=lambda json: [CFGetModResponse(**o) for o in json])
+
+    def get_mod_cdn(self, mod_id: int, file_id: int) -> str:
```

Comment:
This smells funny. We need to reduce the number of API calls in total. When performing a config query, do a single multi-project query to the curseforge API and cache the result for processing in the object. We get all the information we need.

Reply:
Will implement a wrapper for the `/mods/files` endpoint in favor of this wrapper as that would get us the relevant metadata and CDN links of all specified asset files in batches.